Cubical/Structures/Set/CMon/PList.agda

```diff
@@ -1,5 +1,200 @@
 {-# OPTIONS --cubical #-}
 
+-- Definition taken from https://drops.dagstuhl.de/opus/volltexte/2023/18395/pdf/LIPIcs-ITP-2023-20.pdf
 module Cubical.Structures.Set.CMon.PList where
 
--- TODO: QSet
+open import Cubical.Core.Everything
+open import Cubical.Foundations.Everything
+open import Cubical.Data.List as L
+open import Cubical.HITs.PropositionalTruncation as P
+open import Cubical.HITs.SetQuotients as Q
+
+import Cubical.Structures.Set.Mon.Desc as M
+import Cubical.Structures.Set.Mon.List as LM
+import Cubical.Structures.Set.CMon.Desc as M
+import Cubical.Structures.Free as F
+open import Cubical.Structures.Sig
+open import Cubical.Structures.Str public
+open import Cubical.Structures.Tree
+open import Cubical.Structures.Eq
+open import Cubical.Structures.Arity hiding (_/_)
+
+data Perm {ℓ : Level} {A : Type ℓ} : List A -> List A -> Type ℓ where
+  perm-refl : ∀ {xs} -> Perm xs xs
+  perm-swap : ∀ {x y xs ys zs} -> Perm (xs ++ x ∷ y ∷ ys) zs -> Perm (xs ++ y ∷ x ∷ ys) zs 
```

Comment:
Is this the Perm relation from the paper? 

It's also possible to turn comm into a permutation relation, like here: https://github.com/vikraman/generalised-species/blob/master/set/QSet.agda#L18